soccer/gameplay/GameplayModule.cpp

```diff
@@ -26,10 +26,12 @@ using namespace Geometry2d;
 ConfigDouble* GameplayModule::_fieldEdgeInset;
 
 void GameplayModule::createConfiguration(Configuration* cfg) {
-    // this sets the disance from the field boundries to the edge of the global
-    // obstacles, which the
-    // robots will not move through or into
-    _fieldEdgeInset = new ConfigDouble(cfg, "Field Edge Obstacle", .3);
+    /*this sets the disance from the field boundries to the edge of the global
+     * obstacles, which the robots will not move through or into.
+     * The value is given in meters. As of April 20, 2016 the inner 300mm
+     * is free space for the robots.
+     */
+     fieldEdgeInset = new ConfigDouble(cfg, "Field Edge Obstacle", .33);
```

Comment:
GIven that we have an extra 50mm, I thought I would increase the initial value by 50mm. However, the consequences of violating this boundry are now much more severe (high speed coillision) so I wanted to add a bit of safety, I only increased it 30mm. However, it seems to me as if this is still 30mm beyond the actual wall. The last value, .3, was 50mm beyond where the rules state we are allowed to go, but it didn't have a wall to run into. Essentially, should I changed the inset to be more than the value in the rules, exactly equal to it, or less than it to provide some space?